src/Features/CSharp/Portable/CSharpFeaturesResources.resx

```diff
@@ -126,6 +126,9 @@
   <data name="Invert_if_statement" xml:space="preserve">
     <value>Invert if statement</value>
   </data>
+  <data name="Add_define_0" xml:space="preserve">
+    <value>Add #define {0}</value>
```

Comment:
I'm not sure if this should just be Add {0}, as we wouldn't want the `#define` text translated.